NwPluginAPI/Core/Extensions/ItemExtensions.cs

```diff
@@ -0,0 +1,80 @@
+namespace PluginAPI.Core
+{
+	/// <summary>
+	/// Item-related extensions.
+	/// </summary>
+	public static class ItemExtensions
+	{
+		/// <summary>
+		/// Check if an <see cref="T:ItemType">item</see> is an ammo.
+		/// </summary>
+		/// <param name="item">The item to be checked.</param>
+		/// <returns>Returns whether the <see cref="T:ItemType" /> is an ammo or not.</returns>
+		public static bool IsAmmo(this ItemType item)
+		{
+			return item is ItemType.Ammo9x19 or ItemType.Ammo12gauge or ItemType.Ammo44cal or ItemType.Ammo556x45 or ItemType.Ammo762x39;
+		}
+
+		/// <summary>
+		/// Check if an <see cref="T:ItemType">item</see> is a weapon.
+		/// </summary>
+		/// <param name="type">The item to be checked.</param>
+		/// <param name="checkMicro">Indicates whether the MicroHID item should be taken into account or not.</param>
+		/// <param name="checkJailBird">Indicates whether the checkJailBird item should be taken into account or not.</param>
+		/// <returns>Returns whether the <see cref="T:ItemType" /> is a weapon or not.</returns>
```

Comment:
Very good and detailed documentation!

Reply:
:trollface: https://github.com/Exiled-Team/EXILED/blob/12c80de02c6ff45b3b004273237150d303f3b9ed/Exiled.API/Extensions/ItemExtensions.cs#L42

Reply:
Are you saying they didn't write it? I don't really understand what you mean.

Reply:
I do think it might have some similarity, and considering the fact he (PR owner) has had issues writing documentation before, especially the param doc for `checkMicro` which is the only difference between this and the exiled ones.
I mean it's docs and it's essentially just a switch statement for itemType checking I couldn't care less (but it's not my opinion that matters in this case, it's the one of the maintainers).
They would look similar, or very close anyways if you tried writing this a summary from scratch according to the stylecop rules anyways.